internal/graph/resolvers/composite_test.go

```diff
@@ -47,6 +47,16 @@ var (
 	_ generated.CompositeResourceClaimSpecResolver = &compositeResourceClaimSpec{}
 )
 
+func AssertNoErrors(t *testing.T, err error, errs gqlerror.List) {
+	if diff := cmp.Diff(nil, err, test.EquateErrors()); diff != "" {
+		t.Errorf("s.CompositionRef(...): -want error, +got error:\n%s\n", diff)
+	}
+
+	if diff := cmp.Diff(gqlerror.List{}, errs, test.EquateErrors()); diff != "" {
+		t.Errorf("s.CompositionRef(...): -want GraphQL errors, +got GraphQL errors:\n%s\n", diff)
+	}
+}
```

Comment:
This seems a _little_ close to something Go recommends avoiding - https://github.com/golang/go/wiki/TestComments#assert-libraries. That said, it's just a little helper function (not a full library) so I'm not too worried. It may just be the "assert" terminology that caught my attention.

Reply:
It was meant as a helper to keep things more DRY. Would something like this be ok or would you suggest just inlining this code in the 12 places it is used?

```go
func checkErrors(t *testing.T, prefix string, err error, errs gqlerror.List) {
	if diff := cmp.Diff(nil, err, test.EquateErrors()); diff != "" {
		t.Errorf("%s: -want error, +got error:\n%s\n", prefix, diff)
	}

	if diff := cmp.Diff(gqlerror.List{}, errs, test.EquateErrors()); diff != "" {
		t.Errorf("%s: -want GraphQL errors, +got GraphQL errors:\n%s\n", prefix, diff)
	}
}
```

Reply:
I personally typically just inline boilerplate code like this, but I can also see why a helper makes sense - no strong feelings. You will probably want to use https://pkg.go.dev/testing#T.Helper to mark the function as such.

Reply:
I'll go ahead and inline.